flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -98,14 +100,14 @@ protected boolean supportsKeySerializerCheck() {
         return true;
     }
 
-    @Rule public final TemporaryFolder tempFolder = new TemporaryFolder();
+    @TempDir public static File tempFolder;
 
-    @Before
+    @BeforeEach
     public void before() {
```

Comment:
```suggestion
    void before() {
```
Could be package private